offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/affiliation/search/ArchiveAffiliationController.java

```diff
@@ -0,0 +1,37 @@
+package life.qbic.portal.offermanager.components.affiliation.search;
+
+import life.qbic.business.RefactorConverter;
+import life.qbic.business.persons.affiliation.archive.ArchiveAffiliationInput;
+import life.qbic.datamodel.dtos.business.Affiliation;
+
+/**
+ * <b>Holds a reference to the archive affiliation use case</b>
+ *
+ * @since 1.5.0
+ */
+public class ArchiveAffiliationController {
+
+  private ArchiveAffiliationInput archiveAffiliationInput;
+
+  public ArchiveAffiliationController(){}
```

Comment:
I would just go for one type of constructor. You have now two, why is that?

Reply:
Removed